design-documents/graph-ql/coverage/gift-wrapping.md

```diff
@@ -56,25 +61,40 @@ type CartPrices {
     gift_options: GiftOptionsPrices @doc(description: "The list of prices for the selected gift options")
 }
 
-type SalesTotalsInterface {
```

Comment:
Why is this type extension removed?

Reply:
It was removed as part of the customer orders schema.